src/tests/stdlib/switch_user_permission.ab

```diff
@@ -0,0 +1,9 @@
+import { switch_user_permission } from "std"
+// We use `whoami` to get the running user to assign again the same user as permission
+main {
+    unsafe $touch /tmp/amber-symbolic$
+    if switch_user_permission(unsafe $whoami$,"/tmp/amber-symbolic") {
```

Comment:
I got confused when I read `switch_user_permission`. Let's change this function name to `change_owner` or something that implies that `chown` is used in the background